_posts/2021-08-23-gsoc-2021-project.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Google Summer of Code 2021 - Dask Project
+author: Freyam Mehta and Genevieve Buckley
+theme: twitter
+---
+
+{% include JB/setup %}
+
+## Executive Summary
```

Comment:
Overview it is, then

Reply:
I don't have any strong attachment to the phrase - I just saw someone else use it to summarize a dask blogpost and have repeated that pattern ever since.